### PR TITLE
Codebase cleanup

### LINKED ISSUE #610: Fix current linting errors and run autoformatter on entire codebase
From `apps/antalmanac`, run `npm format` and `npm lint`

Over the course of the year we've slowly let formatting and linting issues creep into the project, and it'd be nice to clean them all up before too many accumulate. 

Having un-resolved linter errors will also slow us down if we try to edit any of the files that have unresolved errors since the pre-commit hook will block commits even if the changes you're trying to make aren't the ones that introduced the linter violation.

Also, the `prettier` autoformatter changes sometimes show up in unrelated PRs, which makes it harder to judge the actual scope just based on the specific files and number of lines changed.

### PR DESCRIPTION
**Pull request by KevinWu098**
## Summary
`npm run lint` & `npm run format` were run in `apps/antalmanac`. _🙅‍♂️ tech debt_

## Test Plan
Everything's still clean, changes are mostly minor / syntactical.

## Issues
Closes #610

### CODE REVIEW

File: apps/antalmanac/src/components/AppBar/ImportStudyList.tsx [hunk 1]
```diff
@@ -183,12 +183,13 @@ class ImportStudyList extends PureComponent<ImportStudyListProps, ImportStudyLis
...
-                        { isOpen: false, studyListText: ''}, 
-                        async () => {document.removeEventListener('keydown', this.enterEvent, false)}
-                    )}
+                <Dialog
+                    open={this.state.isOpen}
+                    onClose={() =>
+                        this.setState({ isOpen: false, studyListText: '' }, async () => {
+                            document.removeEventListener('keydown', this.enterEvent, false);
+                        })
+                    }
```
Comment (MinhxNguyen7) [inline apps/antalmanac/src/components/AppBar/ImportStudyList.tsx]: Regardless of what the linter says, I think that this is just less readable. @EricPedley, what do you say should take precedence?

File: apps/antalmanac/src/components/Calendar/CourseCalendarEvent.tsx [hunk 2]
```diff
@@ -117,34 +117,29 @@ interface CourseCalendarEventProps {
...
-        //event.keyCode === 27 reads for the "escape" key
-        if (event.keyCode === 27) {
-          if(paperRef.current)
-                paperRef.current.style.display = 'none';
-        }
-      };
+        const handleKeyDown = (event: { keyCode: number }) => {
+            //event.keyCode === 27 reads for the "escape" key
+            if (event.keyCode === 27) {
+                if (paperRef.current) paperRef.current.style.display = 'none';
```
Comment (MinhxNguyen7) [inline apps/antalmanac/src/components/Calendar/CourseCalendarEvent.tsx]: This should just be one condition. Never nest (if you don't have to)

File: apps/antalmanac/src/components/RightPane/Map/MapMarker.tsx [hunk 3]
```diff
@@ -94,18 +94,19 @@ const MapMarker = ({
...
     }, [markerRef, openPopup, lat, lng, location]);
 
-    function handleKeyPress(event: { key: string; }) {
-        if(event.key === 'Escape' && markerRef.current){
-            //@ts-ignore
+    function handleKeyPress(event: { key: string }) {
+        if (event.key === 'Escape' && markerRef.current) {
+            // eslint-disable-next-line
+            // @ts-ignore
             markerRef.current.leafletElement.closePopup();
```
Comment (MinhxNguyen7) [inline apps/antalmanac/src/components/RightPane/Map/MapMarker.tsx]: This is cursed, but it's being re-written anyway, so it doesn't matter

**Review by MinhxNguyen7 — CHANGES_REQUESTED** [review]
Looks good except for some minor things. I'll have to check that the page actually works the next time around. Also, please fix the merge conflicts so that the staging deploy can run

**Comment by ap0nia** [comment]
The codebase has changed a good amount since this was opened. I'm gonna close this; you can open another PR if you'd like.

**Pull request closed without merging**